pages/item/_itemId/index.vue

```diff
@@ -42,7 +42,7 @@
             </div>
             <v-row
               class="mt-4 align-center"
-              :class="{ 'text-center': !$vuetify.breakpoint.mdAndUp }"
+              :class="{ 'justify-center': !$vuetify.breakpoint.mdAndUp }"
             >
               <v-btn
                 v-if="canPlay(item)"
```

Comment:
Here we're using a v-if which calls to the mixin and a disabled which calls to local method, do these overlap? @MrTimscampi

Reply:
@ThibaultNocchi they're combined, see audio controls for an example

Reply:
You can remove the braces though